CLI/config/config.go

```diff
@@ -0,0 +1,106 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/BurntSushi/toml"
+	flag "github.com/spf13/pflag"
+)
+
+type globalConfig struct {
+	Conf Config `toml:"OGrEE-CLI"`
+}
+
+type Vardef struct {
+	Name  string
+	Value string
```

Comment:
I think Value should be an interface{} type, if we want to pass numerical variables etc